src/Chapter1.hs

```diff
@@ -520,7 +522,10 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+--closestToZero x y = min (abs(x)) (abs(y))
```

Comment:
Almost does a job, but would return the absolute value instead of the initial value of `x` or `y` 🙂

Reply:
Good point. Hadn't thought of that! My bad.